control-plane/src/index.ts

```diff
@@ -218,3 +11,5 @@ process.on("unhandledRejection", err => {
   logger.error("Unhandled rejection", { error: err });
   hdx?.recordException(err);
 });
+
+require("./bootstrap");
```

Comment:
Without this, the server silently fails on errors that are synchronously encountered on `import`s, since we're not typechecking on `tsx`.